_posts/2018-11-02-project-minimalistic-documentation.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: Project documentation 
+description: "Minimalistic but useful project documentation."
+author: bogdan
+modified: 2014-12-24
+tags: [documentation, agile, process, qa]
+categories: [Process]
+image:
+    feature: posts/project_documentation/title.jpg
+---
+
+We don’t like to write documentation. But we do like good documentation for tools we use. Which is a little bit controversial, isn’t it? Actually, this mean that we are too lazy to write docs, but we understand that it’s necessary for any good project. Here is few tips  how to create a “minimalistic but useful” documentation  from guy that don’t like to write documentation too. Hope it will be helpful.
+
+ <!-- more -->
+ 
+## THE PROBLEM
+It’s hard to determine level of detalization for docs. Too many details  and documentation will become outdated almost immediately. Too little detail - and it will be useless. So, what should be covered, and what is not? We can figure it out with a five right questions.
+
+## “WHAT”, “WHY”, “WHO”, “HOW” AND “HOW TO”
+### WHAT
+
+First of all, write down what is this. It should be clear and short - so anyone who will see it at first, will understand what is all about.  For example: *“Configuration.py is a library for configuration management in python apps.”*. 
```

Comment:
First of all, write down what this is _(what the subject is_). It should be clear and concise, so anyone who sees it for the first time is able to understand what it is all about.  For example: *“Configuration.py is a library for configuration management in python apps.”